scss/custom/_wysiwyg.scss

```diff
@@ -0,0 +1,9 @@
+.wysiwyg {
+	p,
+	dd,
+	blockquote,
+	ul,
+	ol {
+		margin: 0;
```

Comment:
We might have to change quite a few things for the wysiwyg content as line-heights are not normalized, lists inherit their padding etc. I will take care of this when needed.